custom_components/sat/sensor.py

```
@@ -245,3 +246,17 @@ def native_value(self) -> str:
     @property
     def unique_id(self) -> str:
         return f"{self._config_entry.data.get(CONF_NAME).lower()}-manufacturer"
+
+
+class SatBoilerSensor(SatEntity, SensorEntity):
+    @property
+    def name(self) -> str:
+        return f"Boiler Status"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)